src/components/header/headerElement.test.js

```diff
@@ -18,9 +18,11 @@ describe('HeaderElement', () => {
   beforeEach(() => {
     const mockInputProps = {
       setActiveDialog: () => { },
+      account: {},
     };
     propsMock = sinon.mock(mockInputProps);
-    wrapper = shallow(<HeaderElement setActiveDialog={mockInputProps.setActiveDialog} />);
+    wrapper = shallow(<HeaderElement account={mockInputProps.account}
+      setActiveDialog={mockInputProps.setActiveDialog} />);
```

Comment:
can be simplified to `wrapper = shallow(<HeaderElement { ...mockInputProps }/>);`